source/OutfitInfoDisplay.cpp

```diff
@@ -459,9 +459,13 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	}
 
 	bool isContinuous = (reload <= 1);
+	double burstReload = outfit.BurstReload();
```

Comment:
```suggestion
```

Might as well use `outfit.BurstReload()` directly below instead of having this variable.

Reply:
In both places? (If we assume that burst reloads less than 1 aren't a thing, or should be rounded up to 1 for calculating up time, it would only be one place.)

Reply:
You're currently only using it in one place anyways. But yeah calling the function twice is fine.

Reply:
Oh, I'm just calling the function in the second place anyway.